packages/@pollyjs/adapter-puppeteer/src/index.js

```diff
@@ -223,6 +223,19 @@ export default class PuppeteerAdapter extends Adapter {
     if (this[LISTENERS].has(target)) {
       const listeners = this[LISTENERS].get(target);
 
+      // puppeteer remove prependListener after v4.0.0, add it if missing
+      if (!target.prependListener) {
```

Comment:
Do we also need to handle `removeListener`?

Reply:
We don't need 。`removeListener` has not been removed

https://github.com/puppeteer/puppeteer/blob/09fca442f701e09929120e1b8abac8e49495ab2d/src/common/EventEmitter.ts#L75